src/components/Todos/index.js

```diff
@@ -26,20 +38,54 @@ function amendStateWithChildren(DOMSource) {
   };
 }
 
+// THE TODOS FUNCTION
+// This is the Todos component which is being exported below.
+// The sources that have been passed to `run` as well are immediately
+// deconstructed into the variables `DOM`, `hashchange`, `initialHash` & `storage`.
 function Todos({DOM, hashchange, initialHash, storage}) {
+  // THE LOCALSTORAGE STREAM
+  // Here we create a localStorage stream that only streams
+  // the first value received from localStorage in order to
```

Comment:
I would replace `received` with `read` here in order to emphasize "sources" == "reads".